bundles/org.openhab.binding.knx/README.md

```diff
@@ -9,14 +9,27 @@ The KNX binding then can communicate directly with this gateway.
 Alternatively, a PC running [KNXD](https://github.com/knxd/knxd) (free open source component software) can be put in between which then acts as a broker allowing multiple client to connect to the same gateway.
 Since the protocol is identical, the KNX binding can also communicate with it transparently.
 
+***Attention:*** With the introduction of UoM support (see `number` channel below) some data types have changed:
+
+- the data type for DPT 5.001 (Percent 8bit, 0 -> 100%) has changed from `PercentType` to `QuantityType`for `number` channels (`dimmer`, `color`, `rollershutter` channels stay with `PercentType`)
+- the data type for DPT 5.004 (Percent 8bit, 0 -> 255%) has changed from `PercentType` to `QuantityType`
+- the data type for DPT 6.001 (Percent 8bit -128 -> 127%) has changed from `PercentType` to `QuantityType`
+- the data type for DPT 9.007 (Humidity) has changed from `PercentType` to `QuantityType`
+
+Rules that check for or compare states and transformations that expect a raw value might need adjustments.
```

Comment:
I'm no expert in English comma rules, but this sentence seems a little complex to read without any. 🙂